docs/source/kfp.rst

```diff
@@ -0,0 +1,16 @@
+.. _kfp-ref:
+
+kfp package
+===========
+
+
+.. toctree::
+    :maxdepth: 2
+
+    kfp.compiler
+    kfp.components
+    kfp.components.structures
```

Comment:
Why showing structures if it is not the user facing interface?

Reply:
I'd like to keep them.
We already refer to the component spec structures in other docs as they're helpful for writing component definitions.

These types already have documentation strings, so they're helpful.

Reply:
OK. Removed the `kfp.components.structures` from the ToC for now.